metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -14,6 +15,12 @@
 logger = logging.Logger(__name__)
 
 
+class TagOption(str, Enum):
+    with_lineage = "with_lineage"
+    without_lineage = "without_lineage"
+    skip = "skip"
```

Comment:
I was unsure if it makes sense to use the Snowflake terminology, "lineage", which might get confusing in the Datahub context, or use the term "propagation" which is not the official term, but sometimes used in the documentation. Also the term "tag inheritance" is used sometimes. [Docs](https://docs.snowflake.com/en/user-guide/object-tagging.html#tag-lineage).

Let me know which term makes most sense.

Reply:
This looks good to me.